custom_components/test/fuzzing/mutation/test_mut_on_helpers.py

```diff
@@ -185,3 +177,60 @@ def test_get_miniserver_type() -> None:
     logger.info("test_get_miniserver_type() test finished.")
 
     assert result["failed_tests"] == 0
+
+
+@pytest.mark.skipif(True, reason="Waiting for Grammar!")
```

Comment:
Waiting for the new grammar to be merged. We then can merge this as a complete feature.